src/lib.rs

```diff
@@ -18,6 +18,7 @@ mod macos;
 #[cfg(target_os = "macos")]
 use macos as platform;
 
+#[derive(Clone, Copy, Debug)]
```

Comment:
I'd rather not put `Copy` on `GlConfig` in case it needs a non-`Copy` field at some point, but otherwise this looks fine to me.

Reply:
Fixed.